website/pages/en/tutorials.js

```diff
@@ -245,6 +230,21 @@ class Tutorials extends React.Component {
         hrefFrom: "baseUrl",
         href: "tutorials/substrate-frontier-workshop/",
       },
+      {
```

Comment:
@jimmychu0807  is there a specific reason for putting this down here?
IMHO as it's an easy one and relevant to builders putting it up higher in the list is a good idea..

Reply:
Those tutorials we put at the top are those deemed essential in starting with Substrate, e.g Starting a Chain, building a basic PoE dApp. This tutorial connecting to grafana and showing dashboard data is "relatively peripheral" on Substrate compared to tutorials above, at least not the first few tuts that new-comers should care about.